upgrade/UPGRADE-v9.1.1-dev.md

```diff
@@ -53,3 +53,15 @@ There you can find links to upgrade notes for other versions too.
 - allow multiple elasticsearch hosts ([#2240](https://github.com/shopsys/shopsys/pull/2240))
     - now it's possible to set multiple elasticsearch hosts like `'["elasticsearch:9200", "elasticsearch2:9200"]'`
     - `Elasticsearch\ClientBuilder` is now created with a different factory, you may want to check your overridden service definition (see PR for details)
+
+- replace deprecated functionality ([#2233](https://github.com/shopsys/shopsys/pull/2233))
+    - replace method by throwing specific exception
+        - `Shopsys\FrameworkBundle\Component\Elasticsearch\Exception\ElasticsearchIndexException::indexAlreadyExists()`
+            - you should throw `Shopsys\FrameworkBundle\Component\Elasticsearch\Exception\ElasticsearchIndexAlreadyExistsException` directly
+    - replace usage of console command and phing targets
+        - usage of command `shopsys:elasticsearch:indexes-create` must be replaced by `shopsys:elasticsearch:indexes-migrate`
```

Comment:
does it make sense to mention the class with command is deprecated and will be removed?

Reply:
It does. You can use this command directly.